src/client/tile.cpp

```diff
@@ -1735,6 +1735,41 @@ bool TextureSource::generateImagePart(std::string part_of_name,
 			baseimg->drop();
 			baseimg = image;
 		}
+		/*
+			[opacity:R
+			Makes the base image transparent according to the given ratio.
+			R must be between 0 and 255.
+			0 means totally transparent.
+			255 means totally opaque.
+		*/
+		else if (str_starts_with(part_of_name, "[opacity:"))
+		{
+			if (baseimg == NULL) {
+				errorstream<<"generateImagePart(): baseimg == NULL "
+						<<"for part_of_name=\""<<part_of_name
+						<<"\", cancelling."<<std::endl;
```

Comment:
`"foo " << "bar.." << std::endl;` Please add spaces around the shifting operators.